localization/en-us.lua

```diff
@@ -411,16 +434,17 @@ return {
             b_Bakery_double_tags = "Double Tags",
             -- KEEP_LITE
             k_bakerycharm = "Charm",
-            k_BakeryCharmInfo = { "Only one charm may be equipped at a time,",
-                "purchasing a new charm replaces an old one." },
+            k_BakeryCharmInfo = { "Only one Charm may be equipped at a time.",
+                "Purchasing a new Charm replaces an old one." },
             b_Bakery_equip = "EQUIP",
             b_Bakery_ante = "(Ante)"
         },
         -- END_KEEP_LITE
         v_dictionary = {
             b_Bakery_deposit = "DEPOSIT $#1#",
             b_Bakery_ante_times = "(Ante*#1#)",
-            v_Bakery_artist = "Artist: #1#"
+            v_Bakery_artist = "Artist: #1#",
+            v_Bakery_coder = "Coder: #1#"
```

Comment:
Ideally, there'd also be a credit for design/idea.

Reply:
[3xCredits](https://github.com/AuroraKy/3xCredits) only has 3 (hence the name) and it's popular enough to prove that it's comprehensive. I'm mostly not using it since I want a solution that can omit credits to myself (since I'm the primary author).

Reply:
Then this calls for a checkbox in a configuration tab.

Reply:
I've implemented a `Made by` badge in b7a38a2 for when it's all one person.